herbert_core/herbert_init.m

```diff
@@ -76,11 +76,9 @@ function addgenpath_message (varargin)
     try
         addpath (genpath_special(string),'-frozen');
     catch ME
-        herbert_off
```

Comment:
as there is now no functionality in the catch, you could remove the try/catch/rethrow